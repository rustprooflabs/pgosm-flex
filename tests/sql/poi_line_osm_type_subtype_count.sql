SELECT osm_type COLLATE "C", osm_subtype COLLATE "C", COUNT(*)
    FROM osm.poi_line
    GROUP BY osm_type COLLATE "C", osm_subtype COLLATE "C"
    ORDER BY osm_type COLLATE "C", osm_subtype COLLATE "C"
;