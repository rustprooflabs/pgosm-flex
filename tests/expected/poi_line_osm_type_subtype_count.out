amenity|bench|5
amenity|parking|2
building|yes|1
historic|yes|5
leisure|slipway|2
man_made|pier|185
